src/main/webapp/pages/common/baseTemplate.jsp

```diff
@@ -170,7 +170,7 @@ if (document.layers) {
 %>
 
 <title>
-		<c:out value="${title}" />
```

Comment:
why are you removing this tittle place holder, don't we have a way to look at where the tittle is coming from and we change from there ?